### PR TITLE
fix(sql): added 'us' units to alter table set param commitLag

### PR DESCRIPTION
**Pull request by bluestreak01**

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 11 / 11 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlUtil.java|10|10|100.00%|
|:large_blue_circle:|io/questdb/cairo/TxnScoreboard.java|1|1|100.00%|

File: core/src/test/java/io/questdb/griffin/AlterTableCommitLagTest.java [hunk 1]
```diff
@@ -239,85 +267,102 @@ public boolean rename(LPSZ from, LPSZ to) {
...
-
-                @Override
-                public long openRO(LPSZ path) {
-                    if (Chars.endsWith(path, TableUtils.META_FILE_NAME) && attempt++ == 1) {
-                        return -1;
-                    }
-                    return super.openRO(path);
+            try (TableReader ignored = engine.getReader(AllowAllCairoSecurityContext.INSTANCE, "X")) {
+                try {
+                    compiler.compile("ALTER TABLE X SET PARAM vommitLag = 111s", sqlExecutionContext);
```
Comment (bsmth) [inline core/src/test/java/io/questdb/griffin/AlterTableCommitLagTest.java:293]: typo

**Review by bsmth — APPROVED** [review]
LGTM 👍🏻

File: core/src/test/java/io/questdb/griffin/AlterTableCommitLagTest.java (hunk 1 above)
Comment (bluestreak01) [reply core/src/test/java/io/questdb/griffin/AlterTableCommitLagTest.java:293]: this typo is intentional, we assert that incorrect parameter name is handled gracefully :)

**Pull request merged**